projects/tinkerbell/tinkerbell-chart/chart/values.yaml

```diff
@@ -26,12 +26,13 @@ boots:
 hegel:
   deploy: true
   name: hegel
-  image: quay.io/tinkerbell/hegel:latest
+  image: public.ecr.aws/eks-anywhere/tinkerbell/hegel:7b286fdc8e8fa91a6e9a179a5494b6ee29fce17b-eks-a-12
   imagePullPolicy: IfNotPresent
+  replicas: 1
   port:
-    hostPortEnabled: true
+    hostPortEnabled: false
     hostPort: 50061
-  args: []
+  args: ["--grpc-use-tls=false"]
```

Comment:
why was this needed?

Reply:
We override this value from the CLI so it shouldn't matter anyways
https://github.com/aws/eks-anywhere/blob/main/pkg/providers/tinkerbell/stack/stack.go#L154